web/packages/teleterm/src/ui/ClusterConnect/ClusterLogin/ClusterLogin.tsx

```diff
@@ -119,6 +122,9 @@ export function ClusterLoginPresentation({
             clearLoginAttempt={clearLoginAttempt}
             shouldPromptSsoStatus={shouldPromptSsoStatus}
             passwordlessLoginState={passwordlessLoginState}
+            shouldSkipVersionCheck={shouldSkipVersionCheck}
+            disableVersionCheck={disableVersionCheck}
+            platform={platform}
```

Comment:
There's a fair share of prop drilling, but this is because `useClusterLogin` doesn't use a context and the login modal is written in the old style.